source/guides/clear/telemetrics.rst

```diff
@@ -495,7 +498,7 @@ Prerequisites
 -------------
 
 Confirm that the telemetrics header file is located on the system at
-:file:`usr/include/telemetry.h`  The `latest version`_ of the file can also be
+:file:`usr/include/telemetry.h`. The `latest version`_ of the file can also be
 found on github for reference, but installing the :command:`telemetry` bundle
```

Comment:
I know I have the bad habit to use telemetry and telemetrics words interchangeably, however since this is docs and not code comments, maybe using the name of the bundle `telemetrics` is more accurate here.

Reply:
Good point!  I'll fix that and the other typo you pointed out.